app/gui/src/project-view/components/GraphEditor/GraphNodeMessage.vue

```diff
@@ -1,16 +1,44 @@
 <script setup lang="ts">
 import SvgButton from '@/components/SvgButton.vue'
 import SvgIcon from '@/components/SvgIcon.vue'
+import { useGraphStore } from '@/stores/graph'
+import { QualifiedImport } from '@/stores/graph/imports'
 import type { Icon } from '@/util/iconName'
+import { QualifiedName } from '@/util/qualifiedName'
+
+const graph = useGraphStore()
 
 const props = defineProps<{
   message: string
   type: MessageType
 }>()
 
+function containsLibraryName(): string | null {
+  const prefix = 'Compile error: Fully qualified name references a library '
```

Comment:
Isn't this message somewhere as constant? Seems a bit dangerous to hardcode it like this.

Reply:
It may be a constant, but in Java code. We don't have a way to share constants between Java and JavaScript. However you are right. This is slightly fragile, let try:

Reply:
Let's _enhance the language server protocol_ we have. For example let's extend [Diagnostic](https://github.com/enso-org/enso/blob/develop/docs/language-server/protocol-language-server.md#diagnostic) with possible hints, like:
```json
hint: {
  type: "import",
  add: "Standard.Table"
}
```
that way we would formalize this API - would be better in the long run then depending on the text of the message.